### PR TITLE
rqt_reconfigure: fix accessing attribute superseded in Qt5

### PR DESCRIPTION
**Pull request by ojura**
Fixes this issue:
https://github.com/ros-visualization/rqt_common_plugins/issues/369

### CODE REVIEW

**Comment by dirk-thomas** [comment]
`QPalette` has a `window` function in Qt 4(.8) as well as Qt 5:
* http://doc.qt.io/qt-4.8/qpalette.html#window
* http://doc.qt.io/qt-5/qpalette.html#window

I am not sure if the old code worked in Qt 4 but I would assume the code in the try block should work for both.

**Comment by cottsay** [comment]
The latest documentation I can see which has `background()` defined is Qt 4.2, and even then it is specifically deprecated in favor of `window()`.

How far back are we targeting support for?

**Comment by dirk-thomas** [comment]
The latest this has to support is Trusty which has 4.8. So I think just changing it to `window()` is sufficient.

File: rqt_reconfigure/src/rqt_reconfigure/paramedit_widget.py [hunk 1]
```diff
@@ -109,15 +109,9 @@ def show_reconf(self, dynreconf_widget):
...
-                [self.palette().window().color().lighter(125),
-                self.palette().window().color().darker(125)]) # Qt5
-        except AttributeError:
-            LayoutUtil.alternate_color(self._dynreconf_clients.itervalues(),
-                [self.palette().background().color().lighter(125),
-                self.palette().background().color().darker(125)]) # Qt4
-
+        LayoutUtil.alternate_color(self._dynreconf_clients.itervalues(),
+            [self.palette().window().color().lighter(125),
+            self.palette().window().color().darker(125)]) # Qt5
```
Comment (cottsay) [inline rqt_reconfigure/src/rqt_reconfigure/paramedit_widget.py]: I think we should remove the reference to Qt5 - we've established that `window()` is used in Qt4 as well.

**Comment by ojura** [comment]
Okay, this should be it.

**Comment by cottsay** [comment]
LGTM

**Comment by dirk-thomas** [comment]
Thanks, the patch itself look good. Can you please either keep the indentation as-is or use the following since your variation doesn't follow PEP8:

```
        LayoutUtil.alternate_color(
            self._dynreconf_clients.itervalues(),
            [self.palette().window().color().lighter(125),
             self.palette().window().color().darker(125)])
```

**Comment by ojura** [comment]
Done.

**Comment by dirk-thomas** [comment]
Thanks for iterating on this.

**Pull request merged**